packages/design-system/src/components/table/components/tableBody/bodyCell.tsx

```diff
@@ -49,6 +99,38 @@ const BodyCell = ({
       } cursor-default flex-1`}
     >
       {cell}
+      <>
+        {isExtension &&
```

Comment:
Let's not make this feature an extension-only thing.

Reply:
Resolved this in latest commits